src/layout/Cell.js

```diff
@@ -41,7 +41,7 @@
         var context = this;
         var offsetMultiple;
 
-        var dragHandles = element.selectAll(".dragHandle").data(this._dragHandles, function (d) { return d; });
+        var dragHandles = element.selectAll("#"+this.id()+" > .dragHandle").data(this._dragHandles, function (d) { return d; });
```

Comment:
We should be using this pattern more often - do you think the following would be worthwhile:
```
Widget.prototype.selectChildren = function(selector) {
    return this._element.selectAll("#" + this.id() + " > " + selector);
};
```
And then for completeness:
```
Widget.prototype.selectAll = function(selector) {
    return this._element.selectAll(selector);
};
```